data

```diff
@@ -1 +1 @@
-Subproject commit bdc5311b6d7a22c445ddcc80143a5fc21368836b
+Subproject commit 0fccfd981388dc8e9c28e963d0f32a18b6e6f08b
```

Comment:
Please open an appropriate pull request on SuperTux/data if needed. Otherwise, I don't know if this is a valid change.